src/modules/Theme/Service.php

```diff
@@ -410,7 +410,7 @@ private function _loadTheme($theme, $client = true, $mod = null)
 
     public function getCurrentRouteTheme(): string
     {
-        if (str_starts_with($_SERVER['REQUEST_URI'], ADMIN_PREFIX)) {
+        if (str_starts_with($this->di['request']->get('_url'), ADMIN_PREFIX)) {
```

Comment:
I don't think we should be relying on `_GET` params given the long term goal of dropping these and routing. Perhaps more URL flexibility should be a longer term feature, rather than patching up what is broken?